From 5b18e618375b098061ce3d781db9b49a1f12dfa7 Mon Sep 17 00:00:00 2001
From: Michael Glembotzki <Michael.Glembotzki@iris-sensing.com>
Date: Thu, 25 Jul 2024 11:37:11 +0200
Subject: [PATCH] Use ext4load instead of fatload partitions

We are using ext4 instead of vfat partitions from now on.

Signed-off-by: Michael Glembotzki <Michael.Glembotzki@iris-sensing.com>
---
 include/configs/imx93_evk.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/imx93_evk.h b/include/configs/imx93_evk.h
index 06c7a8315b9..f98f62427fa 100644
--- a/include/configs/imx93_evk.h
+++ b/include/configs/imx93_evk.h
@@ -90,7 +90,7 @@
 	"loadfdt=fatload mmc ${mmcdev}:${mmcpart} ${fdt_addr_r} ${fdtfile}\0" \
 	"fit_img=fitImage.signed\0" \
 	"fit_addr=0x88000000\0" \
-	"fitboot=fatload mmc ${mmcdev}#${mmcpart} ${fit_addr} ${fit_img}; run mmcargs; bootm ${fit_addr}; reset;\0" \
+	"fitboot=ext4load mmc ${mmcdev}#${mmcpart} ${fit_addr} ${fit_img}; run mmcargs; bootm ${fit_addr}; reset;\0" \
 	"firmware=0\0" \
 	"upgrade_available=0\0" \
 	"ustate=0\0" \
-- 
2.46.0

